kineticTypo/kineticTypo1/Tile.pde

```diff
@@ -1,102 +1,100 @@
 class Tile {
-
- float tx;
- float ty;
   
- int tilesX; 
- int tilesY;
+  // Since we configure this at the beginning we can make this
+  // variable a final static variable, this means we know 
+  // that this variable will never change
+  private final static int fraqX = 4; 
+  private final static int fraqY = 6;
+  private final static int tolleranceX = 30;
  
- float tileW; 
- float tileH;
-
- float fraqX; 
- float fraqY;
+  // declaring this variables as private makes sure we 
+  // can only access this variables within the Tile class
+  private int offsetX;
  
- float fraqW;
- float fraqH; 
+  private int tileW; 
+  private int tileH;
  
- Tile (){ 
-   
-   tx = 0;
-   ty = 0;
+  private int fraqW;
+  private int fraqH; 
+  
+  private int posX;
+  private int posY;
+  
+  private float mx = 0;
+  private float my = 0;
+  
+  private PImage originalImage; 
+  private PGraphics pg;
```

Comment:
All other variables are `private` because we don't use any of them outside the instance of the class itself (yet).